### ISSUE TITLE
Luis build failed: An application with the same name("name") already exist

### ISSUE THREAD

**Issue by Symon94** [microsoft/BotFramework-Composer#9503]
## Describe the bug
Hi to everyone, I have a problem with the Bot Composer. 
As I wrote in the title, when I try to build my bot, I have this error: "_Luis build failed: An application with the same name("name") already exist_" (where "name" is invented).
My bot has multiple languages, so when I build the bot from composer, in the LUIS portal I have an app that is created for all the languages (Example: if the bot have italian, english and spanish languages , in the LUIS portal I'll find 3 app for each languages and dialogs).
I can fix this error deleting the various app that i find in the LUIS portal and then re-build the bot from the composer (it's not an optimal solution), but, if in a second moment I make changes on the composer and build the bot, the error occur again. 
This error is random, once it gives me error for one application, if I do a second build the error it gives me is for another application (once I have this: "Luis build failed: An application with the same name("name") already exist", next time I have this: "Luis build failed: An application with the same name("other_name") already exist").
Has anyone already encountered this problem?

## Version
I'm using the 2.1.2 version of the composer

## Browser
- [X] Electron distribution

## OS
- [X] Windows

## Screenshots
![ScreenError](https://user-images.githubusercontent.com/124156685/216560143-11a71572-3fdd-498b-ad40-307883605197.png)

**Comment by anishprasad01** [comment]
Hi @Symon94,

To create multilingual bots in Composer, you will need to use the [Orchestrator](https://learn.microsoft.com/en-us/composer/concept-orchestrator#multi-lingual). By default, you cannot link multiple LUIS models to a single Composer bot. There is also [documentation](https://learn.microsoft.com/en-us/composer/how-to-create-orchestrator-bot) for creating an Orchestrator bot with Composer.

Additionally, the Microsoft Bot Framework team prefers that **how-to** questions be submitted on [Stack Overflow](http://stackoverflow.com/search?q=%23botframework). The official [Bot Framework Github](https://github.com/Microsoft/BotFramework-SDK/) repo is the preferred platform for information on submitting **bug fixes** and **feature requests**.

This issue will be closed, but if you have a bug or feature request, please feel free to open a new issue in the appropriate repository.

**Comment by Symon94** [comment]
@anishprasad01 I resolved the problem.
I found a bug in Bot Composer, precisely, when the Composer makes this API call:
the problem here is that in this API call the max number of app that the API return is 100 but if you have more than that, the Composer try to create the ones that missing and then return the error "An application with the same name("name") already exist" because in the LUIS portal that app exists.
The first screenshot is the API calls for all the app
The second screenshot represent a search doing with notepad++ of how many app the response JSON of the API call returns. As we can see, the count is 100. 

![Screenshot (30)](https://user-images.githubusercontent.com/124156685/221830272-121f1657-8306-4f8d-a191-99a8b15a0ad4.png)
![Screenshot (31)](https://user-images.githubusercontent.com/124156685/221830284-6822461d-78ec-4113-bf42-b379d5f1c1ab.png)
